crates/ruff/tests/integration_test.rs

```diff
@@ -1214,6 +1298,9 @@ fn redirect_direct() {
     exit_code: 1
     ----- stdout -----
     -:1:1: RUF950 Hey this is a test rule that was redirected from another.
+     |
+     |
```

Comment:
This looks like a bug but it isn't. Or, it is a bug in the test rule that always emits an empty range instead of emitting an actual range.